docs/library/spi_engine/instruction-format.rst

```diff
@@ -60,6 +60,12 @@ Chip-select Instruction
 The chip-select instruction updates the value chip-select output signal of the
 SPI Engine execution module.
 
+The actual values on each pin might be inverted relative to the command
+according to the mask set by :ref:`spi_engine cs-invert-mask-instruction`. The
+Invert Mask acts only on the output registers of the Chip-Select pins. Thus, if
+the last 8 bits of this instruction are 0xFE, CS[0] will be active regardless of
```

Comment:
Wouldn't 0xFE invert all outputs except CS[0]?

Reply:
This text is for the Chip-Select instruction, not the CS Invert Mask Instruction. The idea is that you use the Chip Select instruction the same way, for any Invert Mask state. The values are only inverted at the output. So 0xFE will always activate CS[0]. If the Invert Mask is the default 0x00, then only CS[0] will be low. If the mask is 0xFF, then only CS[0] will be high. If the mask is 0x01, then all CS will be high (but only CS[0] is active, since it's the only one that's inverted)


I think I'll add these examples to the description.

Reply:
This text is for the Chip-Select instruction, not the CS Invert Mask Instruction. The idea is that you use the Chip Select instruction the same way, for any Invert Mask state. The values are only inverted at the output. So 0xFE will always activate CS[0]. If the Invert Mask is the default 0x00, then only CS[0] will be low. If the mask is 0xFF, then only CS[0] will be high. If the mask is 0x01, then all CS will be high (but only CS[0] is active, since it's the only one that's inverted)

I think I'll add these examples to the description.

Reply:
Ah, ok. It just says "this instruction" and the last instruction mentioned is the invert instruction, so it sounds like "this" is referring to the invert instruction. So this could be made more clear that is is referring to a different instruction.

Reply:
You are absolutely right, could be more clear. I tried to make it less ambiguos on the commit below